exist-core/pom.xml

```diff
@@ -456,17 +456,9 @@
         </dependency>
 
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <scope>compile</scope>
-            <!-- scope>test</scope --> <!-- not just test scope, as needed for org.exist.test -->
-            <exclusions>
-                <exclusion>
-                    <!-- we ourselves have a test scoped dependency on a newer hamcrest... see below! -->
-                    <groupId>org.hamcrest</groupId>
-                    <artifactId>hamcrest-core</artifactId>
-                </exclusion>
-            </exclusions>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <scope>test</scope>
```

Comment:
I think this scope needs to be compile - see the comment before. If so, can you reinstate the comment too please?